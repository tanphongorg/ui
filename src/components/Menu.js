import React, { Component } from 'react'
import { Input, Menu, Icon } from 'semantic-ui-react'
import { Link } from 'react-router-dom'

export default class MenuExampleSecondary extends Component {
  render () {
    const { className } = this.props
    return (
      <Menu secondary className={className}>
        <Menu.Item name='home'>
          <Link to='/'><Icon name='home' /></Link>
        </Menu.Item>
        <Menu.Item name='about'>
          <Link to='/about'>About</Link>
        </Menu.Item>
        <Menu.Menu position='right'>
          <Menu.Item>
            <Input icon='search' placeholder='Search...' />
          </Menu.Item>
          <Menu.Item name='logout' />
        </Menu.Menu>
      </Menu>
    )
  }
}
