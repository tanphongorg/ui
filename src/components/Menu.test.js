/* global it , expect */

import React from 'react'
import Menu from './Menu'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

it('renders without crashing', () => {
  const component = shallow(<Menu />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
