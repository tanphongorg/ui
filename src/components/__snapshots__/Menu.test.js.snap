// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Menu
  secondary={true}
>
  <MenuItem
    name="home"
  >
    <Link
      replace={false}
      to="/"
    >
      <Icon
        as="i"
        name="home"
      />
    </Link>
  </MenuItem>
  <MenuItem
    name="about"
  >
    <Link
      replace={false}
      to="/about"
    >
      About
    </Link>
  </MenuItem>
  <MenuMenu
    position="right"
  >
    <MenuItem>
      <Input
        icon="search"
        placeholder="Search..."
        type="text"
      />
    </MenuItem>
    <MenuItem
      name="logout"
    />
  </MenuMenu>
</Menu>
`;
